test/ktn_lists_SUITE.erl

```diff
@@ -45,10 +46,20 @@ end_per_suite(Config) ->
 -spec delete_first(config()) -> ok.
 delete_first(_Config) ->
     Fun = fun(N) -> 0 == N rem 2 end,
-    
+
     [] = ktn_lists:delete_first(Fun, []),
     [] = ktn_lists:delete_first(Fun, [4]),
     [4] = ktn_lists:delete_first(Fun, [4, 4]),
     [1, 3] = ktn_lists:delete_first(Fun, [1, 3]),
     [1, 3] = ktn_lists:delete_first(Fun, [1, 4, 3]),
     [1, 3, 4] = ktn_lists:delete_first(Fun, [1, 4, 3, 4]).
+
+-spec split_when(config()) -> ok.
+split_when(_Config) ->
+    IsDot = fun(Ch) -> $. == Ch end,
+
+    ["{a}.", " {b}."] = ktn_lists:split_when(IsDot, "{a}. {b}."),
+    [] = ktn_lists:split_when(IsDot, ""),
+    ["."] = ktn_lists:split_when(IsDot, "."),
+    ["{a}.", " {b}.", "{c, d, e}"] = ktn_lists:split_when(IsDot, "{a}. {b}.{c, d, e}"),
```

Comment:
Line 64 is too long:     ["{a}.", " {b}.", "{c, d, e}"] = ktn_lists:split_when(IsDot, "{a}. {b}.{c, d, e}"),.